content/docs/iac/guides/building-extending/providers/implementers/python.md

```diff
@@ -539,25 +539,25 @@ During preview, some values are unknown:
 def _is_unknown(self, value):
     """Check if a value is unknown (not yet computed)."""
     if isinstance(value, dict):
-        return value.get('4dabf18193072939515e22adb298388d') == 'unknown'
+        return value.get('4dabf18193072939515e22adb298388d') == '04da6b54-80e4-46f7-96ec-b56ff0331ba9'
     return False
 ```
 
 Handle unknowns in your Diff implementation by skipping comparison for unknown values.
 
 ## Packaging for distribution
 
-To distribute your provider, you need to create a wrapper script that Pulumi can execute. Create a shell script named `pulumi-resource-myfiles`:
+Package the provider in a .tar.gz file:
 
 ```bash
-#!/bin/bash
-DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-cd "$DIR"
-source venv/bin/activate
-exec python -m provider "$@"
+tar -zcf myfiles.tar.gz __main__.py provider/__init__.py provider/server.py PulumiPlugin.yaml requirements.txt schema.json
 ```
 
-Make it executable with `chmod +x pulumi-resource-myfiles`. This script activates the virtual environment and runs the provider module. Package everything including the `venv` directory into a distributable archive, or install dependencies system-wide and adjust the script accordingly.
+This plugin can then be installed as a regular pulumi resource plugin:
+
+```bash
+pulumi plugin install resource myfiles --file ./myfiles.tar.gz 0.0.1
+```
 
 See [Publishing packages](/docs/iac/guides/building-extending/packages/publishing-packages/) for full details.
```

Comment:
Do we have a friendlier guide to packaging and SDK generation somewhere?

Something that roughly covers:

* Upload your .tar.gz somewhere with the expected filename for os/arch, etc. Note this is kinda dumb for Python, but I think this is geared towards binary providers
* Set PluginDownloadURL in the schema with the download root
* Generate language SDKs
* Publish language SDKs


Alternatively, we could provide instructions for a local SDK here instead. These only require making the provider source available somewhere, bypassing the tedium of SDK publishing.

Reply:
For now I updated this to link out to the publishing docs and to the local SDK docs.